submissions/phpslonuk/Popup/css/style.css

```diff
@@ -0,0 +1,222 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+/* Navigation */
+
+.nav {
+  position: fixed;
+  top: 10px;
+  right: 100px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+}
+
+.nav_item {
+  margin: 10px;
+}
+
+.nav_item label{
+	cursor: pointer;
+}
+
+.nav_item:hover {
+  outline-style: none;
+  border-color: gray;
+}
+
+.nav_item_link {
+  padding-top: 5px;
+}
+
+.nav_item_mail, .nav_item_images {
+  font-size: 17px;
+  text-decoration: none;
+  color: #767676;
+}
+
+.nav_item_img {
+  width: 20px;
+  height: 20px;
+  margin-top: 3px;
+  border-radius: 10px;
+}
+
+#trigger {
```

Comment:
good practice to avoid id-selectors for styles. Generally, ID-selector can be used for JS